internal/cli/atlas/backup/exports/jobs/jobs.go

```diff
@@ -21,10 +21,11 @@ import (
 
 func Builder() *cobra.Command {
 	const use = "jobs"
+	deprecated := append([]string{}, cli.GenerateAliases("bucketId")...)
 	cmd := &cobra.Command{
 		Use:     use,
 		Short:   "Manage cloud backup export jobs for your project.",
-		Aliases: cli.GenerateAliases(use),
+		Aliases: cli.GenerateAliases("jobId", deprecated...),
```

Comment:
here you are giving the job command the alias `jobId` which (I think) is not what you want. What you may want to do here is to add a new flag `JobID` and deprecate the existing `bucketId` flag